catalog-template.yml

```
@@ -1232,26 +1244,30 @@ redis:
         service: "aws-broker"
     - id: "3nd227bf-sm4c-44c1-0b81-575fel666443"
       name: "redis-3node-large"
-      description: "AWS Elasticache Redis 5.0.6 Three node"
+      description: "AWS Elasticache Redis 6.2 Three node"
```

```suggestion
      description: "AWS Elasticache Redis Three node"
```

Seems like we're not including the version in other plan descriptions since it is now configurable